python/pyxbos/pyxbos/drivers/hvac/hvac.py

```diff
@@ -0,0 +1,162 @@
+from pyxbos.driver import *
+from pyxbos import hvac_pb2
+import pymortar
+import BAC0
+import re
+import os,sys
+import json
+import time
+
+class HVACDriver(Driver):
+    def setup(self, cfg):
+        # A dictionary mapping point names to their classes
+        self.pymortar_client = pymortar.Client({
+            'username': cfg['mortar_api_username'],
+            'password': cfg['mortar_api_password']
+        })
+        self.bacnet = BAC0.connect()
+        self.point_class_map = get_point_class_mapping(self.pymortar_client, cfg['building'])
+        self.devices = [BAC0.device(dev[2], dev[3], self.bacnet) for dev in self.bacnet.devices]
+
+    def read(self, requestid=None):
+        for dev in self.devices:
+            # A dictionary mapping point names to their bacnet values
+            point_value_map = get_point_value_mapping(dev.points)
+
+            #AHU-1.DPR-O Damper point MISSING from Bacnet points
+
+            output = {}
+            for equipclass in self.point_class_map:
+                if equipclass not in ['Boiler', 'Chilled_Water_Pump', "Hot_Water_Pump"]:
+                    if equipclass not in output:
+                        output[equipclass] = {}
+
```

Comment:
Can you write some documentation explaining how the parsing logic works? It seems like some of this is hardcoded to the particular test building, but ideally this is logic that can generalize to more buildings. I realize this is difficult without having another set of points that we can use, but its worth starting to make that effort now in anticipation of future changes

Reply:
I just made an excluded_equipment list in get_point_class_dict()